servers/rendering/renderer_rd/renderer_scene_render_rd.cpp

```diff
@@ -1985,25 +411,25 @@ Ref<Image> RendererSceneRenderRD::sky_bake_panorama(RID p_sky, float p_energy, b
 }
 
 void RendererSceneRenderRD::_update_dirty_skys() {
-	Sky *sky = dirty_sky_list;
+	RendererSceneSkyRD::Sky *skyrd = sky.dirty_sky_list;
```

Comment:
I would reverse the naming of skyrd and sky

Right now ``sky`` is referencing the RendererSceneSkyRD object while ``skyrd`` is referencing a ``Sky`` object. IMO it makes more sense to use ``sky`` for the ``Sky`` object and ``sky_rd`` for the RendererSceneSkyRD object.

Reply:
Yup the skyrd is temporary, I'm moving all this code into `RendererSceneSkyRD` so this line of code will simply become:
`RendererSceneSkyRD::Sky *sky = dirty_sky_list;`

I just had to temporary rename it while the code lives in `RendererSceneRenderRD`

Reply:
These should now all be correctly named.